src/linker/Linker/Driver.cs

```diff
@@ -1086,15 +1086,18 @@ static void Usage ()
 
 			Console.WriteLine ();
 			Console.WriteLine ("Actions");
-			Console.WriteLine ("  -c ACTION           Sets action for all framework assemblies. Defaults to 'link'");
-			Console.WriteLine ("                        copy: Analyze whole assembly and save it to the output");
-			Console.WriteLine ("                        copyused: Same as copy but only for assemblies which are needed");
-			Console.WriteLine ("                        link: Remove any unused IL or metadata and optimizes the assembly");
-			Console.WriteLine ("                        skip: Do not process the assembly");
-			Console.WriteLine ("                        addbypassngen: Add BypassNGenAttribute to unused methods");
-			Console.WriteLine ("                        addbypassngenused: Same as addbypassngen but unused assemblies are removed");
-			Console.WriteLine ("  -u ACTION           Sets action for any user assembly. Defaults to 'link'");
-			Console.WriteLine ("  -p ACTION ASM       Overrides the default action for specific assembly name");
+			Console.WriteLine ("  --trim-action ACTION      Sets action for assemblies annotated as trimmable. Defaults to 'link'");
+			Console.WriteLine ("                              copy: Analyze whole assembly and save it to the output");
+			Console.WriteLine ("                              copyused: Same as copy but only for assemblies which are needed");
+			Console.WriteLine ("                              link: Remove any unused IL or metadata and optimizes the assembly");
+			Console.WriteLine ("                              skip: Do not process the assembly");
+			Console.WriteLine ("                              addbypassngen: Add BypassNGenAttribute to unused methods");
+			Console.WriteLine ("                              addbypassngenused: Same as addbypassngen but unused assemblies are removed");
+			// https://github.com/mono/linker/blob/main/docs/design/trimmed-assemblies.md#assemblymetadataistrimmable-false
+			// If we add support for opting out by setting "IsTrimmable" to "False", we should add an option to control behavior for such assemblies.
+			// Console.WriteLine ("  --no-trim-action ACTION   Sets action for assemblies annotated as non-trimmable. Defaults to 'copy'");
+			Console.WriteLine ("  --default-action ACTION   Sets action for assemblies that have no trimmability annotation. Defaults to 'link'");
+			Console.WriteLine ("  --action ACTION ASM       Overrides the default action for specific assembly name");
```

Comment:
Do we need two options? What about simpler

```
-action ACTION       Sets action for all assemblies that have no IsTrimmable attribute. Defaults to 'link'
-action ACTION ASM   Overrides the default action for the specific assembly name
```

Reply:
I took your naming suggestions. It's a little odd because `--action ACTION ASM` will override not just the `--action ACTION` for unattributed assemblies, but also the `--trim-mode ACTION` for ones with IsTrimmable. In other words, this might give the impression that there's only one "default" action, when really there are two - the `--trim-mode` and the `--action`. That's why I initially picked the names `--trim-action` and `--default-action`.

However I do like the consistency of your suggestion. Either way we will need to think carefully about the names on the SDK side.